src/test/java/org/reaktivity/nukleus/amqp/internal/streams/server/AmqpServerIT.java

```diff
@@ -1266,4 +1266,33 @@ public void shouldCloseConnectionWhenMaxFrameSizeExceededWithMultipleSessions()
     {
         k3po.finish();
     }
+
+    @Test
+    @Specification({
+        "${route}/server/controller",
+        "${client}/connection/server.idle.timeout.expires/client" })
+    @Configure(name = "nukleus.amqp.idle.timeout", value = "10000")
```

Comment:
Recommend reducing these timeouts significantly to avoid making the build take a lot of extra time.